doc/conf.py

```diff
@@ -28,6 +28,7 @@ def _workaround_imports_typechecking():
     """Load some packages needed for type annotations."""
     import ipywidgets  # noqa
     import pandas  # noqa
+    import pint  # noqa
```

Comment:
this was needed to fix the import error in sphinx builds @marscher @vhirtham

Reply:
This is so confusing.. At first I added this workaround for pandas only. Eventually we have a module name collision with the stdlib or something.

Reply:
it only happens in sphinx though for whatever reason...